resources/dicts/patrols/plains/hunting/leaf-bare.json

```diff
@@ -1574,5 +1574,87 @@
                 "prey": ["very_small"]
             }
         ]
+    },
+    [
+        {
+            "patrol_id": "pln_hunt_watchandlearnfox1",
+            "biome": ["plains"],
+            "season": ["leaf-bare"],
+            "types": ["hunting"],
+            "tags": [
+                "hunting",
+                "respect",
+                "trust",
+                "platonic",
+                "patrol_to_s_c",
+                "patrol_to_r_c"
+            ],
+            "patrol_art": "mtn_hunt_leaffallsnowstorm",
+            "patrol_art_clean": null,
+            "min_cats": 3,
+            "max_cats": 6,
+            "min_max_status": {
+                "deputy": [3, 6],
+                "warrior": [3, 6],
+                "leader": [3,6],
+                "all apprentices": [3, 6]
+            } 
+        },
+        {"weight": 20,
+            "chance_of_success": 30,
+            "relationship_constraint": null,
+            "pl_skill_constraint": null,
+            "intro_text": "Leaf-bare is taking its toll. Your patrol is cold, frusterated, and most of all, hungry. r_c notices something odd. Is that a fox in the distance? Why is it jumping into the snow like that?",
+            "decline_text": "First the cold, and the snow, and now a fox? This is too much for one day, p_l decides. Back to camp, another patrol can try again later. ",
```

Comment:
is this meant to be p_l or r_c ? This might be a different cat than the one that saw the fox

Reply:
In my mind, I thought the implication was the patrol was speaking to each other so if the cats were different it wouldn't be confusing. Do you think it would be better if it were all just s_c?